internal/worker/worker.go

```diff
@@ -303,8 +304,9 @@ func (worker *Worker) tryCreateStandby(ctx context.Context) {
 
 	worker.logger.Debugf("warming-up the standby instance")
 
+	warmUpTimeout := time.Duration(worker.standbyParameters.Warmup.TimeoutSeconds) * time.Second
```

Comment:
This seems to cause NPE in tests: https://cirrus-ci.com/task/5965693838950400?logs=test#L3291-L3303

Reply:
Another point is that since the `worker.standbyParameters` and its inner messages are now `nil`-able, and the standby parameters can be set from the backend, it would be nice if we can check for `nil` here in a more defensive manner.